pack.sh

```diff
@@ -8,6 +8,7 @@ export NODE_OPTIONS="--max-old-space-size=4096 ${NODE_OPTIONS:-}"
 root=$PWD
 
 PACMAK=${PACMAK:-jsii-pacmak}
+ROSETTA=${ROSETTA:-jsii-rosetta}
```

Comment:
Kinda sorta thinking I'd rather those be prefixed with `JSII_`, but not a deal breaker.